nservicebus/azure-service-bus/understanding-transactions-and-delivery-guarantees.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Understanding internal transactions and delivery guarantees
+reviewed: 2016-04-20
+tags:
+- Azure
+- Cloud
+- Transactions
+---
+
+## Understanding internal transactions and delivery guarantees
+
+### Versions 6 and below
```

Comment:
I did split into partials, but hitting a [roadblock](https://builds.particular.net/viewLog.html?buildId=182647&tab=buildResultsDiv&buildTypeId=DocsWeb_ContentVerification) on the build server :(